lib/IRGen/IRGenModule.cpp

```diff
@@ -266,7 +266,7 @@ IRGenModule::IRGenModule(IRGenerator &irgen,
   // A tuple type metadata record has a couple extra fields.
   auto tupleElementTy = createStructType(*this, "swift.tuple_element_type", {
     TypeMetadataPtrTy,      // Metadata *Type;
-    SizeTy                  // size_t Offset;
+    Int32Ty                  // size_t Offset;
```

Comment:
Remove the extra space added here (the comments should still line up).

Is the struct still the same size on x86-64 after this change? I guess the struct should still be pointer aligned because of the `TypeMetadataPtrTy` field?

Reply:
We should probably change `// size_t Offset;` to `// int32_t Offset;` too.